sdk/src/types.rs

```diff
@@ -135,6 +135,19 @@ impl LocalAccount {
         Ok(Self::new(address, key, sequence_number))
     }
 
+    /// Create a new account from the given private key in hex literal.
+    pub fn from_private_key(private_key: &str, sequence_number: u64) -> Result<Self> {
+        let key = AccountKey::from_private_key(Ed25519PrivateKey::try_from(
+            hex::decode(private_key.trim_start_matches("0x"))?.as_ref(),
+        )?);
+
+        Ok(Self {
+            address: key.authentication_key().account_address(),
+            key,
```

Comment:
This needs to be setting `auth`, can you fix it @JmPotato?

Perhaps someone change this while it was in PR

Reply:
Fixed. PTAL.